src/DwitTech.AccountService.Data/Entities/BaseEntity.cs

```diff
@@ -10,7 +10,7 @@ namespace DwitTech.AccountService.Data.Entities
     public abstract class BaseEntity
     {
         public int Id { get; set; }
-        public DateTime CreatedOnUtc { get; } = DateTime.UtcNow;
+        public DateTime CreatedOnUtc { get; private set; } = DateTime.UtcNow;
```

Comment:
Please why did you made set private, I want to learn, please.

Reply:
I did it because as it turns out, using "get" alone as it was before, was preventing the CreatedOnUtc value from being generated in the migration sheet along with other properties for a class that inherits this BaseEntity class. I'm not sure exactly why, but from the little research I did, I think while generating migrations. entity framework only makes use of properties that directly influence the entity in the db. Perhaps, while it was on "get" alone, ef assumed it was pointless to add a property that cannot be set in the db. Adding a "private set" there corrected the issue, by making sure ef recognizes the property as important, as well as preventing the value from being set or changed elsewhere(since we want it fixed). The value in that column can only be retrieved, but not manipulated(unless of course someone with access modifies the column).